Makefile

```diff
@@ -114,6 +114,10 @@ hosts-chromium:
 	            ln -sfv "$(LIB_DIR)/browserpass/hosts/chromium/$(APP_ID).json" "/etc/chromium/native-messaging-hosts/$(APP_ID).json"; \
 	            [ -e "/etc/chromium/native-messaging-hosts/$(APP_ID).json" ] || echo "Error: the symlink points to a non-existent location" >&2; \
 	            ;; \
+	FreeBSD)    mkdir -p "${PREFIX}/etc/chrome/native-messaging-hosts/"; \
```

Comment:
Huh, really, `/etc/chrome` is the folder for Chromium on FreeBSD? Could you confirm that this is not a typo?